app/models/user.rb

```diff
@@ -85,17 +72,42 @@ def can_access_evss?
     edipi.present? && ssn.present? && participant_id.present?
   end
 
+  def edipi
+    select_source_id(:edipi)
+  end
+
+  def icn
+    select_source_id(:icn)
+  end
+
+  def participant_id
+    select_source_id(:vba_corp_id)
+  end
+
+  def mhv_correlation_id
+    mhv_correlation_ids.first
+  end
+
+  def mvi
+    return nil unless loa3?
+    @mvi ||= Mvi.from_user(self).fetch
+  end
+
   private
 
   def mhv_correlation_ids
     return @mhv_correlation_ids if @mhv_correlation_ids
```

Comment:
Yea, this was my bad. I should have done the memoization with `||= begin ... end` to better draw attention to it instead of doing it this way. Not sure why we wouldn't want to memoize that method though.